src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/backend.py

```diff
@@ -83,8 +92,9 @@ def ts_openvino(subgraph, example_inputs):
         om.validate_nodes_and_infer_types()
 
         device = "CPU"
-        if (os.getenv("OPENVINO_TORCH_BACKEND_DEVICE") is not None):
-            device = os.getenv("OPENVINO_TORCH_BACKEND_DEVICE")
+        openvino_torch_backend_device = _get_option("OPENVINO_TORCH_BACKEND_DEVICE", options)
```

Comment:
could you please provide a usage example?

I suppose specifying `OPENVINO_TORCH_BACKEND_DEVICE` as an option does not look good. I would rather use `device` or similar.

Reply:
This feature of having options sent to the backend is only available in Pytorch master right now and not in any publicly available release. So until the next Pytorch release I think we can have both environment variables and options. So i think it would be better if the env variable and options have somewhat similar names

Reply:
So to use these options from the application side, before we can add these options to torch.compile and send them as a map. 
Like this: 
opts = {"OPENVINO_TORCH_MODEL_CACHING" : 1, "OPENVINO_TORCH_BACKEND_DEVICE" : "GPU"}
model = torch.compile(model, backend="openvino", options=opts)

Reply:
Ok, but I suppose OV will be officially integrated into PT in the same release when these options are available in PT publicly? 
So, we can rely on their availability? In any case, how would ideal final solution look like?

Reply:
> could you please provide a usage example?
> 
> I suppose specifying `OPENVINO_TORCH_BACKEND_DEVICE` as an option does not look good. I would rather use `device` or similar.

@ilya-lavrenov Agree with shorter names, see last commit

Reply:
> This feature of having options sent to the backend is only available in Pytorch master right now and not in any publicly available release. So until the next Pytorch release I think we can have both environment variables and options. So i think it would be better if the env variable and options have somewhat similar names

@suryasidd I think we could mix 2 approaches, see my last commit